### PR TITLE
upgraded dependencies

### PR DESCRIPTION
**Pull request by tylerthehaas**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 */
 var path = require('path'),
     extend = require('deep-extend'),
-    osHomedir = require('os-homedir'),
+    osHomedir = require('os').homedir(),
```
Comment (amadsen) [inline index.js:13]: hmm... this requires deprecation of support for ancient versions of Node that didn't have `os.homedir()`. It still might be the right thing to do.

Comment (amadsen) [reply index.js:13]: https://www.npmjs.com/package/os-homedir is deprecated and `os.homedir()` was introduced in the io.js days. We can make this change.

File: index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ var settings = {
   useEnv: true,
   usePkg: true,
   cwd: process.cwd(),
-  home: osHomedir(),
+  home: osHomedir,
```
Comment (amadsen) [inline index.js:27]: I'll might change this back to calling `os.homedir()` every time. It likely makes no difference for the vast majority of cases (config usually is processed once at app startup).

**Review by amadsen — DISMISSED** [review]
Approved. I'll merge. Just as a note, I'd have appreciated a description in the PR explaining why the code changes needed to happen. It only took a little research, but it could have saved me some time.

**Comment by tylerthehaas** [comment]
yes I should've been more diligent about explaining the changes. sorry! I was using node version 12.16.2 when I got a passing test run. I can try with 13.11.0 to see what the errors were and how we can fix them.

**Comment by tylerthehaas** [comment]
Just tried it with node v13.11.0 and got a successful test run locally.

**Comment by amadsen** [comment]
Thank you for confirming tests pass for you. I have not figured out why they are not working for me, though it seems to be related to the `pkg-up` module somehow.

**Comment by tylerthehaas** [comment]
I’m assuming you’ve reinstalled node_modules right?

**Review by amadsen — APPROVED** [review]
This is embarrassing. I forgot to run `npm ci` before running tests on the branch. Approving and merging.

**Comment by amadsen** [comment]
> I’m assuming you’ve reinstalled node_modules right?

Nope. I just realized that this morning, approved, merged, then saw this. Silly mistake. Sorry.

**Comment by tylerthehaas** [comment]
easy to do.

**Comment by amadsen** [comment]
Changes have been published as `1.0.0-rc.0`. If they work well for you and me we can republish it as `1.0.0`.

**Pull request merged**